### PR TITLE
Editor: Fix remove function of UIPoints.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: see #20981

**Description**

Ensures that adding/removing points works without runtime errors.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev41/editor/index.html

File: editor/js/libs/ui.js [hunk 1]
```diff
@@ -124,6 +124,12 @@ UIElement.prototype = {
 
 		return this;
 
+	},
+
+	getIndexOfChild: function ( element ) {
+
+		return Array.prototype.indexOf.call( this.dom.children, element.dom );
```
Comment (Mugen87) [inline editor/js/libs/ui.js:131]: see https://stackoverflow.com/questions/222841/most-efficient-way-to-convert-an-htmlcollection-to-an-array

**Comment by Mugen87** [comment]
Review: https://github.com/mrdoob/three.js/pull/20981#discussion_r553662516

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**